server/controllers/card.js

```diff
@@ -0,0 +1,77 @@
+const asyncHandler = require("express-async-handler");
+const Card = require("../models/Card");
+const { v4: uuidv4, v4 } = require("uuid");
+
+exports.getCards = asyncHandler(async (req, res) => {
+  const Cards = await Card.find();
```

Comment:
To follow JavaScript conventions, variable names generally shouldn't be uppercase.

Reply:
Careless use of replaceAll on my part, thanks!